new-log-viewer/src/services/decoders/ClpIrDecoder.ts

```diff
@@ -31,19 +34,38 @@ class ClpIrDecoder implements Decoder {
         return this.#streamReader.getNumEventsBuffered();
     }
 
-    buildIdx (beginIdx: number, endIdx: number): Nullable<LogEventCount> {
+    // eslint-disable-next-line class-methods-use-this
+    getFilteredLogEventMap (): FilteredLogEventMap {
+        // eslint-disable-next-line no-warning-comments
+        // TODO: Update this after log level filtering is implemented in clp-ffi-js
+        return null;
```

Comment:
Would it be better if we make the log-level filtering a no-op rather than completely disabling the decoder?

```suggestion
        console.error("Not implemented.");
        return Array.from({length: this.#streamReader.getNumEventsBuffered()}, (_, i) => i);
```